### ISSUE TITLE
Shouldn't 'destroy' reset the 'firstLoad' flag?

### ISSUE THREAD

**Issue by vishnurk** [lukehaas/Scrollify#389]
I am attempting to destroy an existing Scrollify instance before re-initializing. On re-initializing, the 'afterRender' callback is NOT triggered. Took a closer look at code and noticed this -
`      if(firstLoad===true) {
        settings.afterRender();
        firstLoad = false;
      }`
Shouldn't the call to destroy reset the value of firstLoad to TRUE? I do see other variables getting reset.

**Comment by lukehaas** [comment]
Yes, looks like the destroy method should probably set `firstLoad` back to true.
If you raise a PR for this, I'll be happy to merge it in.
